transcom/mymove
# INT-B-20472

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A980088&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Is there anything you would like reviewers to give additional scrutiny?


### How to test

1. Login to milmovelocal
2. Create a new move
3. Before submitting move click the 'Cancel Move' button
4. Check database for the canceled move and it's shipments
5. Create a new move
6. Submit move to SC 
7. Cancel Move button should be gone
